test/byContract/IexecMaintenance/IexecMaintenance.test.ts

```diff
@@ -211,8 +211,9 @@ describe('Maintenance', async () => {
     }
 });
 
-async function hashDomain(domain: IexecLibOrders_v5.EIP712DomainStructOutput) {
-    return ethers.utils._TypedDataEncoder.hashDomain({
+//TODO: Move to utils
+export async function hashDomain(domain: IexecLibOrders_v5.EIP712DomainStructOutput) {
+    return ethers.TypedDataEncoder.hashDomain({
```

Comment:
```suggestion
    return TypedDataEncoder.hashDomain({
```
other option: directly import `TypedDataEncoder` module

Reply:
Sure see [Import from ethers](https://github.com/iExecBlockchainComputing/PoCo/pull/189/commits/01a96ab3e20eece28b60f32d3a08f1f93304bd37) (same comment everywhere)